src/styles/App.scss

```diff
@@ -135,3 +135,17 @@ a {
     margin: 12px;
     text-decoration: underline;
 }
+.home {
+    &__page {
```

Comment:
dont need the split: there are going to be no home class. its a good catch tho it should be home_page not home__page